tests/test_sbt.py

```diff
@@ -138,7 +138,7 @@ def test_tree_v1_load():
                         leaf_loader=SigLeaf.load)
 
     testdata1 = utils.get_test_data(utils.SIG_FILES[0])
-    to_search = next(signature.load_signatures(testdata1))
+    to_search = next(load_signatures(testdata1))
```

Comment:
since this is being replaced, can also do 
```suggestion
    to_search = load_one_signature(testdata1)
```
in all these tests